opensds/nbp
# [CSI] Removal of host devices during volume detach

**What this PR does / why we need it**:
When volume is getting detached, the corresponding host detach was not performed by csi plugin
This was resulting in accumulation of stale entries

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
This PR fixes : https://github.com/opensds/nbp/issues/332

**Special notes for your reviewer**:

> Any other PR(s) this PR is dependant on: None

Test Steps:
1) Create a pod during which volume will be attached
2)Delete the pod
3) Verify device disk repository and ensure that created entries are removed

![image](https://user-images.githubusercontent.com/30930862/72787604-40c92700-3c56-11ea-9924-ecf051b11ae9.png)





**Release note**:

```release-note
```
